### DISCUSSION TITLE
What is the allowed time for the presentation ?

### DISCUSSION THREAD

**Discussion by lucaslattion** [heig-vd-dai-course/heig-vd-dai-course#102] [Q&A] [answered]
# What is the allowed time for the presentation ?

<html>
<body>

22 | Everyone speaks during the presentation, and the presentation lasts the time allowed | 0.2
-- | -- | --

</body>
</html>

**Comment by ludelafo** [comment, marked as answer]
Hello,

I intended to notify all students tomorrow morning but I will do it now so everyone has the information.

@heig-vd-dai-course/students,

The allowed time is **5 minutes**.

You can find the order of passage in the conclusion to part 1 here: <https://github.com/heig-vd-dai-course/heig-vd-dai-course/tree/main/07-conclusion-to-part-1>.

Please let me know if you have any other questions.

Have a nice evening!

Copy: @heig-vd-dai-course/teaching-staff
